src/index.js

```diff
@@ -19,21 +19,23 @@ export default function (options = {}, callback = function noop () {}) {
   let testAttempt = 1
   let parsedOptions = parseOptions(options)
   let parser = getParser(parsedOptions.parser)
+  let plainLogger = new Logger(null)
```

Comment:
A good idea. Implemented this.

Messages generated by flake use `logger.log()` while messages generated by protractor use 'logger.protractor()`.